### PR TITLE
*: rename a doc

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

- rename `online-ddl-scheme.md` to `feature-online-ddl-scheme.md`
- minor edit `task-configuration-file-full.md`

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):https://github.com/pingcap/docs-dm/pull/143

### CODE REVIEW

File: zh/task-configuration-file-full.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ target-database:                # 下游数据库实例配置
   port: 4000
   user: "root"
   password: "/Q7B9DizNLLTTfiZHv9WoEAKamfpIUs="  # 推荐使用经 dmctl 加密后的密码
-  session:                                      # 设置 TiDB 的 session 变量，在 v1.0.6 版本引入。更多变量及解释参见 https://docs.pingcap.com/zh/tidb/stable/tidb-specific-system-variables
+  session:                                      # 设置 TiDB 的 session 变量，在 v1.0.6 版本引入。更多变量及解释参见 `https://docs.pingcap.com/zh/tidb/stable/tidb-specific-system-variables`
```
Comment (ran-huang) [inline zh/task-configuration-file-full.md:48]:
```suggestion
  session:                                      # 设置 TiDB 的 session 变量，在 v1.0.6 版本引入。更多变量及解释参见 `https://docs.pingcap.com/zh/tidb/stable/system-variables`
```
`tidb-specific-system-variables.md` is merged into `system-variables.md`

**Comment by yikeke** [comment]
PTAL again @ran-huang

**Comment by ran-huang** [comment]
LGTM

**Pull request merged**